src/boot/paging_32.zig

```diff
@@ -0,0 +1,31 @@
+const zasm = @import("zasm");
+
+extern var p4_table: [4096]u8;
+
+export fn enable_paging() void {
+    // Load P4 to CR3 register. (CPU uses this to access the P4 table).
+    const raw_p4_addr = @intCast(u64, @ptrToInt(&p4_table[0]));
+    var p4_addr = zasm.PhysAddr.initUnchecked(raw_p4_addr);
```

Comment:
Why not `const`?

Reply:
> Why not const?

Done d85d018ace010d14524c18319cca7d500fcd4af9.